src/main/java/org/embulk/base/restclient/JacksonServiceResponseSchema.java

```diff
@@ -18,11 +18,20 @@
 import org.embulk.spi.type.Type;
 import org.embulk.spi.type.Types;
 
+import org.embulk.base.restclient.writer.BooleanColumnWriter;
+import org.embulk.base.restclient.writer.ColumnWriter;
+import org.embulk.base.restclient.writer.DoubleColumnWriter;
+import org.embulk.base.restclient.writer.JsonColumnWriter;
+import org.embulk.base.restclient.writer.LongColumnWriter;
+import org.embulk.base.restclient.writer.SchemaWriter;
+import org.embulk.base.restclient.writer.StringColumnWriter;
+import org.embulk.base.restclient.writer.TimestampColumnWriter;
+
 import static org.embulk.spi.Exec.newConfigSource;
 
-public class SchemaWriterFactory
+public class JacksonServiceResponseSchema
```

Comment:
The name intended a schema of responses from the target service. The purpose of the class is a "schema" which maps the service's JSON to Embulk's schema.